### PR TITLE
[Doc Improvement][Message extension GCC GCCH updates][2445150]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/bot-features.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b73fd78e6c91c92d715320f103738cc9112953aa/msteams-platform%2Fbots%2Fbot-features.md) | 85 | 19 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/236c603c-e98d-48d6-8da0-40b5be134117) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/bot-features.md](https://github.com/MicrosoftDocs/msteams-docs/blob/53f891ca4084e67958ca8fc59d418e3258f71fbf/msteams-platform%2Fbots%2Fbot-features.md) | 85 | 19 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2a9742bf-7a0a-4955-b7f2-ffd3df0064d1) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/bot-features.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a10fb3d2caf43ca5f43546c4ad75c0fd52a15b7e/msteams-platform%2Fbots%2Fbot-features.md) | 85 | 19 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9928c550-ba47-47e0-b35a-c89fb49a3828) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/bot-features.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4a7c11b66e7f8e220cc62fad69561fda991dfed2/msteams-platform%2Fbots%2Fbot-features.md) | 84 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0f5e1ed7-8051-49f3-8ebe-e4ff141c8fe0) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/what-are-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4a7c11b66e7f8e220cc62fad69561fda991dfed2/msteams-platform%2Fbots%2Fwhat-are-bots.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/23ff3215-75cd-4613-8f48-8a0f2c99ff92) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by v-npaladugu** [comment]
PR is an extension to update comments received in PR [5853](https://github.com/MicrosoftDocs/msteams-docs/pull/5853) after publishing the PR.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/bot-features.md](https://github.com/MicrosoftDocs/msteams-docs/blob/08a2f79f93c624ed2cd205dafecfeb45cedaae88/msteams-platform%2Fbots%2Fbot-features.md) | 89 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/27385edf-4715-421e-be04-db78cf8a3dac) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/what-are-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/08a2f79f93c624ed2cd205dafecfeb45cedaae88/msteams-platform%2Fbots%2Fwhat-are-bots.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/464e5bbe-5b55-4189-9610-2e714151a4e2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/bot-features.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f74b858df67a289faf8846d1f184791f720adc25/msteams-platform%2Fbots%2Fbot-features.md) | 92 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e84abb55-e9ab-4c81-bf75-c55228f10182) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/what-are-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f74b858df67a289faf8846d1f184791f720adc25/msteams-platform%2Fbots%2Fwhat-are-bots.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dafc7840-81db-4fa6-8d9d-685640d60fb5) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/what-are-bots.md [hunk 1]
```diff
@@ -12,8 +12,10 @@ A bot is also referred to as a chatbot or conversational bot. It is an app that
 
 > [!IMPORTANT]
 > Currently, bots are available in Government Community Cloud (GCC) and GCC-High but not in Department of Defense (DOD).
-> 
-> Bot applications within Microsoft Teams are available in GCC-High through [Azure bot Service](/azure/bot-service/channel-connect-teams).
+>
+> Bot applications within Microsoft Teams are available in GCC-High through [Azure bot Service](/azure/bot-service/how-to-deploy-gov-cloud-high) and bot channel registration should be done in Azure Government portal.
+>
+>Bots in GCCH only support up to manifest version v1.10. Image URL's in Adaptive Cards are not supported in GCCH environment. You can replace an image URL with Base64 encoded DataUri.
```
Comment (juna-gupta) [inline msteams-platform/bots/what-are-bots.md:18]: This limitation is for apps in GCCH. (irrespective of the app using bot capabilities)

File: msteams-platform/bots/what-are-bots.md [hunk 2]
```diff
@@ -12,8 +12,10 @@ A bot is also referred to as a chatbot or conversational bot. It is an app that
 
 > [!IMPORTANT]
 > Currently, bots are available in Government Community Cloud (GCC) and GCC-High but not in Department of Defense (DOD).
-> 
-> Bot applications within Microsoft Teams are available in GCC-High through [Azure bot Service](/azure/bot-service/channel-connect-teams).
+>
+> Bot applications within Microsoft Teams are available in GCC-High through [Azure bot Service](/azure/bot-service/how-to-deploy-gov-cloud-high) and bot channel registration should be done in Azure Government portal.
```
Comment (juna-gupta) [inline msteams-platform/bots/what-are-bots.md:16]: We should also talk about or add a link to consent modeling in GCCH. ie tenants are responsible for enabling LOB applications and how the data is processed / where it is stored.

File: msteams-platform/bots/bot-features.md [hunk 3]
```diff
@@ -48,105 +47,6 @@ The [Bot Framework](https://dev.botframework.com/) is a rich SDK used to create
 
 Webhooks and connectors connect your bot to your web services. Using webhooks and connectors, you can create a bot for basic interaction, such as creating a workflow or other simple commands. They're available only in the team where you create them and are intended for simple processes specific to your company's workflow. For more information, see [what are webhooks and connectors](~/webhooks-and-connectors/what-are-webhooks-and-connectors.md).
```
Comment (juna-gupta) [inline msteams-platform/bots/bot-features.md:48]: Webhook and connectors are not available in GCCH. Do we need to document that here or we cover it someplace else?

**Review by juna-gupta — CHANGES_REQUESTED** [review]
Left some comments for doc improvements.

File: msteams-platform/bots/bot-features.md (hunk 3 above)
Comment (Shawn-Massoud-MS) [reply msteams-platform/bots/bot-features.md:48]: This is Mentioned in another location. Sorry for the delay in messages.

File: msteams-platform/bots/bot-features.md [hunk 4]
```diff
@@ -48,17 +48,16 @@ The [Bot Framework](https://dev.botframework.com/) is a rich SDK used to create
...
-
-> [!NOTE]
-> * Bots in GCCH only support  up to manifest version v1.10.
-> * Image URL's in Adaptive Cards are not supported in GCCH environment. You can replace an image URL with Base64 encoded DataUri.
-> * Bot channel registration in Azure Government will provision web app bot, app service (app service plan), and application insights also but it doesn't support to provision the azure bot service only (no app service).
+> Bot applications within Microsoft Teams are available in GCC-High through [Azure bot Service](/azure/bot-service/channel-connect-teams) and bot channel registration should be done in Azure Government portal.
+>
+> * Bots in GCC-High only support up to manifest version v1.10.
+> * Image URLs in Adaptive Cards are not supported in GCCH environment. You can replace an image URL with Base64 encoded DataUri.
+> * Bot channel registration in Azure Government will provision web app bot, app service (app service plan), and Application Insights also but it doesn't support to provisioning the Azure Bot Service only (no app service).
```
[the remark this answers is not in the document]
Comment (Shawn-Massoud-MS) [reply msteams-platform/bots/bot-features.md]: They Documentation has been published @v-npaladugu can we remove this part docs?

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/bot-features.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8e785195538d5dc7346694d935df81979852e79e/msteams-platform%2Fbots%2Fbot-features.md) | 92 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9383270e-19b2-4cf9-ba9b-32c0694dcad7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/what-are-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8e785195538d5dc7346694d935df81979852e79e/msteams-platform%2Fbots%2Fwhat-are-bots.md) | 94 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f6a07aab-93e4-4225-a201-6841300fd232) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/bot-features.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fe6b86f2b7f0c48f3c84eece698ac887e03d0d42/msteams-platform%2Fbots%2Fbot-features.md) | 92 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bd936b42-3ecc-47ae-870e-a72e9974b599) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/what-are-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fe6b86f2b7f0c48f3c84eece698ac887e03d0d42/msteams-platform%2Fbots%2Fwhat-are-bots.md) | 94 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/85dfb27e-6b76-4ee7-9fbd-9aec7bd21eb5) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/bot-features.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ed658c95a8aa77cf07f880e0d020eff80c365bdf/msteams-platform%2Fbots%2Fbot-features.md) | 92 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2cd82f03-c1a2-4c3d-8b24-a0c25136a4db) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/what-are-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ed658c95a8aa77cf07f880e0d020eff80c365bdf/msteams-platform%2Fbots%2Fwhat-are-bots.md) | 94 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/484d0ea3-c34f-4ef3-97ac-71f112a3a24f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/bot-features.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d36bbad9a26c330deac4cc8fb7b1ef71d473bc52/msteams-platform%2Fbots%2Fbot-features.md) | 92 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e7f09849-c8c9-4f1d-a8bb-4da70723cf49) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/what-are-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d36bbad9a26c330deac4cc8fb7b1ef71d473bc52/msteams-platform%2Fbots%2Fwhat-are-bots.md) | 94 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/14e3727f-f271-4ca4-a0b9-364c84386c95) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — APPROVED** [review]
Approved

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**